### ISSUE TITLE
Make This Website Mobile Responsive

### ISSUE THREAD

**Issue by mehraTushar** [mehraTushar/React-Practice#2]
Currently this looks good on tabs and desktop but its not mobile friendly
make it responsive
![image](https://github.com/mehraTushar/React-Practice/assets/70595562/a41589b5-89ad-4ed5-8261-38eb56985d67)

**Comment by Rajveer07** [comment]
I would like to do , If you assign it to me

**Comment by mehraTushar** [comment]
yes you can work on it.

**Comment by mehraTushar** [comment]
@Rajveer07 let me know.... if you want to continue with this ticket

**Comment by Rajveer07** [comment]
Yes I am since there is semester exam so it could take 2 or 3 days.

**Comment by mehraTushar** [comment]
@Rajveer07 no worries you can take your time
